gitpod-io/gitpod
# [server] Cleanup (effectively) unused config: stage/KUBE_STAGE

## Description


## Related Issue(s)

Related to this epic: https://github.com/gitpod-io/gitpod/issues/9097

## How to test
 - open a workspace on this branch
 - search the whole repo for these words, and make sure we no longer use them:
   - GITPOD_INSTALLATION_LONGNAME
   - theiaPluginsBucketNameOverride
   - stage (exact case, full word, limit to `*.tsx?` files)

## Release Notes

```release-note
none
```

## Documentation
